### PR TITLE
CSS selectors completed

### PR DESCRIPTION
**Pull request by eliaset096**
The CSS selectors step has been completed.

### CODE REVIEW

**Review by manuelq12 — COMMENTED** [review]
Can you show me the console to see that the execution of the test passed please?

File: cypress/integration/buy-shirt.spec.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      cy.visit("http://automationpractice.com/")
+      cy.get("#block_top_menu > ul > li:nth-child(3) > a").click()
+      cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click()
+      cy.get("[style*=\"display: block;\"] .button-container > a").click()
+      cy.get(".cart_navigation span").click()
+  
+      cy.get("#email").type("aperdomobo@gmail.com")
+      cy.get("#passwd").type("WorkshopProtractor")
+  
+      // Debes completar la prueba ...
```
Comment (garzuzo) [inline cypress/integration/buy-shirt.spec.ts]: remove comment

File: cypress/integration/buy-shirt.spec.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      cy.get("[name=cgv]").click()
+      cy.get("[name=processCarrier]").click()
+      cy.get("[class=bankwire]").click()
+      cy.get("#cart_navigation > button").click()
+
+      cy.get("#center_column > div > p > strong")
+        .should("have.text", "Your order on My Store is complete.")
+    });
+  });
+
```
Comment (garzuzo) [inline cypress/integration/buy-shirt.spec.ts]: add new line at the end of the file

File: cypress/integration/buy-shirt.spec.ts (hunk 1 above)
Comment (eliaset096) [reply cypress/integration/buy-shirt.spec.ts]: comment removed

File: cypress/integration/buy-shirt.spec.ts (hunk 2 above)
Comment (eliaset096) [reply cypress/integration/buy-shirt.spec.ts]: line added.

**Comment by eliaset096** [comment]
Screenshot_1
Screenshot_2
Screenshot_3




> Can you show me the console to see that the execution of the test passed please?

File: cypress/integration/buy-shirt.spec.ts (hunk 2 above)
Comment (holgiosalos) [reply cypress/integration/buy-shirt.spec.ts]: please make sure that the last line is empty, it should not contain any space :)

File: cypress/integration/buy-shirt.spec.ts (hunk 1 above)
Comment (holgiosalos) [reply cypress/integration/buy-shirt.spec.ts]: I still see the comment

File: cypress/integration/buy-shirt.spec.ts (hunk 2 above)
Comment (holgiosalos) [reply cypress/integration/buy-shirt.spec.ts]: When GitHub shows the following icon:
image

Means that the file does not end with just one and only one empty or blank line

File: cypress/integration/buy-shirt.spec.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      cy.get("[name=processAddress]").click()
+      cy.get("[name=cgv]").click()
+      cy.get("[name=processCarrier]").click()
+      cy.get("[class=bankwire]").click()
+      cy.get("#cart_navigation > button").click()
+
+      cy.get("#center_column > div > p > strong")
+        .should("have.text", "Your order on My Store is complete.")
+    });
+  });
```
Comment (garzuzo) [inline cypress/integration/buy-shirt.spec.ts]: EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon:
image

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**